### ISSUE TITLE
[bug] Web interface at dezoomify.ophir.dev doesn't correctly works with HTTP sites (not HTTPS)

### ISSUE THREAD

**Issue by agiudiceandrea** [lovasoa/dezoomify#832]
**Describe the bug**
When used with an HTTP website, the Web interface at dezoomify.ophir.dev tries to download the tiles from a not existent HTTPS site.

**To Reproduce**
Try with http://sast.beniculturali.it/TecaViewerImgPuglia/servlet/iipsrv?FIF=/mnt/Storage/Puglia/ASBA-TRANI/GCCT/PE/V10/F8/UC0001/IIPImage/ASBA-TRANI_GCCT_PE_V10_F8_UC0001_00001.tif (note the HTTP in the URL)

It tries to download from https://sast.beniculturali.it/TecaViewerImgPuglia/servlet/iipsrv?FIF=/mnt/Storage/Puglia/ASBA-TRANI/GCCT/PE/V10/F8/UC0001/IIPImage/ASBA-TRANI_GCCT_PE_V10_F8_UC0001_00001.tif&JTL=7,2 instead (note the wrong HTTPS in the URL instead of the correct HTTP)

**Expected behavior**
Download with HTTP protocol when needed.

**Screenshots**

**Browser:**
 - Google Chrome 131.0.6778.205
 - Mozilla Firefox 133.0.3

**Comment by agiudiceandrea** [comment]
@lovasoa, is there a plan to fix this issue?

**Comment by lovasoa** [comment]
No, it's a security feature implemented by web browsers, not a bug in dezoomify.

**Comment by agiudiceandrea** [comment]
Thanks for the info. Anyway, it seems weird to me that it is possible to browse and visualize (actually, download) zoomified images on a HTTP website, but it is not possible for the scripts to download the images' tiles from such HTTP website.

**Comment by agiudiceandrea** [comment]
Anyway, it looks like it is simple to workaround the issue: just setting to "Allow" the site setting "Insecure content" for https://dezoomify.ophir.dev/.
